docs/requirements.txt

```diff
@@ -1,9 +1,3 @@
-sphinx==6.1.3
-sphinx_rtd_theme==1.2.0
-sphinx-autoapi==2.0.1
-nbsphinx
-ipython
-jupytext
-jupyter
+jupyter-book
 matplotlib
-numpy
\ No newline at end of file
+numpy
```

Comment:
If you already have your docs requirement in your `pyproject.toml`, you probably don't need this file anymore.

Reply:
Okay, thank you for this information. The jupyter-book default setup had this file, so I wasn't sure about it.